backends/backends.c

```diff
@@ -300,6 +300,35 @@ void backend_set_prometheus_variables(int *default_port,
 #endif /* ENABLE_PROMETHEUS_REMOTE_WRITE */
 }
 
+/**
+ * Set MongoDB variables
+ *
+ * Set the variables necessaries to work with this specific backend.
+ *
+ * @param default_port  the default port of the backend
+ * @param brc function called to check the result.
+ * @param brf function called to format the msessage to the backend
+ * @param type the backend string selector.
```

Comment:
I think you forgot to remove the comment for "type" or you did not create this parameter yet.